autoconnect/autoconnect-web/src/dockerflow.rs

```diff
@@ -26,8 +26,19 @@ pub fn config(config: &mut web::ServiceConfig) {
 }
 
 /// Handle the `/health` and `/__heartbeat__` routes
+// note, this changes to `blocks_in_conditions` for 1.76+
+#[allow(clippy::blocks_in_if_conditions)]
```

Comment:
Half of me wonders if this case is even supposed to trigger `blocks_in_if_conditions` whereas half of me also agrees we should just put this entire, longer expression on its own let line instead of quieting clippy (but not going to block on it)

Reply:
I was honestly wondering the same thing, but decided to do this mostly because I found the long form really hard to read in comparison. 

Here, at the point we've gotten the error, we're firing an `error!()` macro. We *could* do something with `.else()` but then getting the error back would be a pain, so you have to a `match` and blargh.